### ISSUE TITLE
ODS-7 not detected

### ISSUE THREAD

**Issue by dewhattens** [openpreserve/odf-validator#62]
File without any cells values not detected

Testing D:\ODS1\TestCorpusV2\ODS-7\WithOutCellValues.ods
APP-1: [INFO] Validating D:\ODS1\TestCorpusV2\ODS-7\WithOutCellValues.ods.
APP-4: [INFO] Validation report for D:\ODS1\TestCorpusV2\ODS-7\WithOutCellValues.ods.
DOC-3: mimetype [INFO] OpenDocument MIMETYPE application/vnd.oasis.opendocument.spreadsheet detected
VALID, no errors, no warnings and 1 info message found.

**Comment by carlwilson** [comment]
This should be fairly straightforward as an empty sheet looks as follows:

```xml
<office:spreadsheet>
    <table:calculation-settings table:automatic-find-labels="false" table:use-regular-expressions="false" table:use-wildcards="true"/>
    <table:table table:name="Sheet1" table:style-name="ta1">
        <table:table-column table:style-name="co1" table:default-cell-style-name="Default"/>
        <table:table-row table:style-name="ro1">
            <table:table-cell/>
        </table:table-row>
    </table:table>
    <table:named-expressions/>
</office:spreadsheet>
```

Remember that a spreadsheet comprises multiple worksheets. What's the policy on empty worksheets in a spreadsheet where other worksheets have content?

**Comment by carlwilson** [comment]
Created new issue for ODF_7: #95
